easybuild/framework/easyblock.py

```diff
@@ -1445,7 +1445,20 @@ def extensions_step(self, fetch=False):
 
     def package_step(self):
```

Comment:
+1 on this, since this is most likely only one way of packaging that we'll support in the (near) future, other candidates may include Docker containers, Nix packages (?), ...

@sntgluca: let me know if you're up for tackling @stdweird's remark